common/history/characters/0_dvg_lotharingia.txt

```diff
@@ -0,0 +1,43 @@
+CHARACTERS = {
+	c:LOT = {
+		
+		
+		create_character = {
+			ruler = yes
+			is_general = yes
+			first_name = "Damien"
+			last_name = "de_Aigle"
+			culture = cu:lotharian
+			female = no
+			commander_rank = default
+			interest_group = ig_armed_forces
+			ideology = ideology_caudillismo

```

Comment:
Can only do leader Ideologies

Reply:
What's this supposed to mean? Caudillismo works just fine when implementing it in the game. Also, not sure what other leader ideology would fit De Aigle's character

Reply:
In victoria 3 there are leader ideologies and party ideologies, caudillismo is a party ideology. It _kinda_ works in the game but it has incorrect flavor text and the affect is weird. I am not sure of what one to use, however.